frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/SelectedNodeInputOutputTab.tsx

```diff
@@ -4,23 +4,34 @@ import { PipelineRunTaskDetails, TaskReferenceMap } from '~/concepts/pipelines/c
 import { getNameAndPathFromTaskRef, getValue } from '~/concepts/pipelines/topology/pipelineUtils';
 import TaskDetailsInputParams from '~/concepts/pipelines/content/pipelinesDetails/taskDetails/TaskDetailsInputParams';
 import TaskDetailsOutputResults from '~/concepts/pipelines/content/pipelinesDetails/taskDetails/TaskDetailsOutputResults';
+import { PipelineRunTaskParam } from '~/k8sTypes';
 
 type SelectedNodeInputOutputTabProps = {
   task: PipelineRunTaskDetails;
   taskReferences: TaskReferenceMap;
+  parameter?: PipelineRunTaskParam[];
 };
 
 const SelectedNodeInputOutputTab: React.FC<SelectedNodeInputOutputTabProps> = ({
   task,
   taskReferences,
+  parameter,
 }) => {
   const params =
     task.params?.map((p) => {
+      const paramValueMatch = p.value.match(/^\$\((params\.([a-zA-Z0-9-_]+))\)/);
+      if (paramValueMatch && parameter) {
+        const paramName = paramValueMatch[1].split('.')[1];
+        const paramFromParameter =
+          parameter.find((result) => result.name === paramName)?.value ?? p.value;
+        return { ...p, value: paramFromParameter };
+      }
       const ref = getNameAndPathFromTaskRef(p.value);
       if (!ref) {
         return p;
       }
       const [name, path] = ref;
+
```

Comment:
Was this blank line intentional to improve readability, or an accident? :) If the former no probs, if the latter, it should be removed.